motor/core.py

```diff
@@ -1795,3 +1796,86 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         pass
+
+class AgnosticClientEncryption(AgnosticBase):
+    __motor_class_name__ = 'MotorClientEncryption'
+    __delegate_class__ = ClientEncryption
+
+    create_data_key = AsyncCommand(doc=create_data_key_doc)
+    encrypt = AsyncCommand()
+    decrypt = AsyncCommand()
+
+    _close = AsyncCommand(attr_name='close')
+
+    def __init__(self, kms_providers, key_vault_namespace, key_vault_client, codec_options, io_loop=None):
+        """Explicit client-side field level encryption.
+
+        The ClientEncryption class encapsulates explicit operations on a key
```

Comment:
Should we copy/paste this whole docstring? Perhaps we should just say:
```
        Takes the same constructor arguments as
        :class:`pymongo.encryption.ClientEncryption`, as well as:

        :Parameters:
          - `io_loop` (optional): Special event loop
            instance to use instead of default
 ```
 
 I would prefer this so that we don't need to duplicate the docs in pymongo and motor.

Reply:
ok, sounds good to me!